Dockerfile

```
@@ -0,0 +1,8 @@
+FROM ubuntu:21.04
+FROM node:14
```

❗️
Тут не нужна убунта и и не нужна полноценная нода
```suggestion
FROM node:14-alpine
```

---

Круто, не знал что такая есть. Спасибо! 👍

---

Не за что ✋🏻
`alpine` — это тег, которым помечают самые минимальные сборки, это касается не только ноды)